### DISCUSSION TITLE
Need help?

### DISCUSSION THREAD

**Discussion by rimutaka** [rust-lang/rust-analyzer#14832] [General]
@Veykril , Lucas, I have a few hours per week to work on something interesting.

Point me at an issue or two that need doing, if you have any?

Maybe helping with the docs is the easiest place to start.

**Comment by Veykril** [comment]
> Point me at an issue or two that need doing, if you have any?

We have way too many :)

If you want to help out with some doc issues you can take a look at the corresponding label https://github.com/rust-lang/rust-analyzer/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3AA-documentation

**Comment by rimutaka** [comment]
@Veykril , these 3 look doable. Which one shall I pick?

* https://github.com/rust-lang/rust-analyzer/issues/13172
* https://github.com/rust-lang/rust-analyzer/issues/9053
* https://github.com/rust-lang/rust-analyzer/issues/13178
